### DISCUSSION TITLE
How to Generate Certificates and use them for running client and server in an arbitary path.

### DISCUSSION THREAD

**Discussion by Cozy03** [open-quantum-safe/oqs-provider#186] [Q&A]
Hi I am new to this library so it might be a silly question. But where are the commands documented for generating public and private keys using pq algorithms like dilithium and falcon and also using kyber. 
Also if I run the server and client in two different PCs(connected via ethernet switch) and want to moniter if TLS connection is actually established via wireshark using a third PC will it be possible? 
I have tried the same using OQS's old library that used version 1.1.1 but didnt see much on wireshark. That is why I am asking..

**Comment by baentsch** [comment]
You can use the exact same `openssl` commands that you'd use for classic crypto.  Check out [the relevant test script](https://github.com/open-quantum-safe/oqs-provider/blob/main/scripts/oqsprovider-certgen.sh) for example code.

Thanks for the question, btw! This just triggered me to note that the current documentation is not sufficiently explicit enough about this. Until #187 lands, you can check the [updated README in the branch](https://github.com/open-quantum-safe/oqs-provider/blob/mb-doc-update/README.md#using).

**Reply by baentsch** [reply]
@Cozy03 With #187 landed, are you OK to close ("Mark as answered") this issue?
